lib/cli/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@common-grants/cli",
-  "version": "0.1.0-alpha.12",
+  "version": "0.1.0",
```

```suggestion
  "version": "0.0.0",
```

Shouldn't this be 0.0.0 based on how we did the first release of the core library?

---

Yes, that was just an oversight. I'll fix it. Good catch!